scripts/adapters/update-min-os-version.rb

```diff
@@ -4,12 +4,16 @@
 require 'json'
 require 'open3'
 
+def sdk_min_os_version
+  return Gem::Version.new('13.0')
+end
+
 # Function to obtain the min OS version info from the partner podspec
 def min_os_version(pod_name, pod_version)
 
   # Update cocoapods repos to ensure the local info is up to date
   `pod repo add-cdn trunk https://cdn.cocoapods.org`
-  `pod repo update`
+  # `pod repo update`
```

Comment:
Is this intentional?

Reply:
Good catch, had this commented out for testing.